<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorCreateDialog</class>
 <widget class="QDialog" name="AuthorCreateDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>79</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Add new user</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="AuthorInfield"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="CancelBtn">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="OkBtn">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>AuthorInfield</tabstop>
  <tabstop>CancelBtn</tabstop>
  <tabstop>OkBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
